package.json

```diff
@@ -134,6 +139,10 @@
     "react-transition-group": "^4.4.1",
     "redux": "^3.7.0",
     "subprocess-middleware": "0.1.0",
-    "underscore": "^1.13.2"
```

Comment:
Should keep this at 10.13.2 (or higher)?

Reply:
@alexkb0009 Yeah, we already upgraded it in master after the dependabot alert. I merged it into the webpack5 branch now.